src/main/scala/org/constellation/ConstellationNode$.scala

```diff
@@ -325,6 +325,8 @@ object ConstellationNode$ extends IOApp with IOApp.WithContext {
   private def validateInitConditions(nodeConfig: NodeConfig): Stream[IO, Unit] = {
     import constellation.PublicKeyExt
 
+    val minCpuCount = sys.env.get("CL_MIN_CPU_COUNT").flatMap(x => Try(x.toInt).toOption).getOrElse(4)
```

Comment:
Why are these changes in this PR all of a sudden? :O

Reply:
I needed them to test this change on a cluster (otherwise the node would not start) so I rebased against my other branch. These changes would disapeear when I would click `merge and rebase`. But to make the PR clean I rebased yet again, this time against dev.